### DISCUSSION TITLE
Disabling password recovery via email

### DISCUSSION THREAD

**Discussion by gloribus** [directus/directus#18091] [Feature Requests]
### Summary

Disabling password recovery via email

### Motivation

To increase security, auto-generated email addresses are used, and recovery is carried out only by a direct request to the manager. But the reset password button confuses users

### Detailed Design

A switch in the project settings that removes the reset password button

### Must Have:
- A Switch in project settings
- B ?

### Should Have:
- C ?

### Could Have:
- D ?

### Won't Have:
- E ?

### Drawbacks

?

### Alternatives

?

### Adoption Strategy

This is a minor visual change and will not affect

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
